tests/sovereign_consumer_upgrade_local/hermes-0.15.0-old-connection/start_provider.sh

```diff
@@ -0,0 +1,134 @@
+#!/bin/bash
```

Comment:
Topic for tomorrow: Lets go over how neccessary it is to merge these shell scripts into the ICS repo